iothub_client/samples/iothub_client_c2d_streaming_proxy_sample/iothub_client_c2d_streaming_proxy_sample.c

```diff
@@ -111,22 +111,29 @@ static void on_bytes_received(void* context, const unsigned char* buffer, size_t
 
     if (g_is_uws_client_ready)
     {
-        size_t send_size;
-
-        while (size > 0)
+        if (buffer == NULL)
         {
-            send_size = size <= MAX_WEBSOCKET_PAYLOAD_SIZE_IN_BYTES ? size : MAX_WEBSOCKET_PAYLOAD_SIZE_IN_BYTES;
+            printf("on_bytes_received invoked with NULL buffer (unexpected)\r\n");
```

Comment:
This is a bit unclear: if this is an error can we say "error" and maybe exit the process?
The code continues here so I assume this is just a warning?

Reply:
Also, if it's a warning, it's unclear if it is a developer issue that should be corrected.

Reply:
Missing `(void)` before `printf`.

Reply:
I agree, we could just exit the process. It is an error if a buffer is null.